src/paste/deploy/config.py

```diff
@@ -155,9 +155,6 @@ def __init__(self, application, config):
     def __call__(self, environ, start_response):
         global wsgilib
         if wsgilib is None:
-            import pkg_resources
-
-            pkg_resources.require('Paste')
```

Comment:
I can’t see the point to this code because the next line will just work or fail. It seems to just be for the error message?